dev/PushNotifications/PushNotificationsLongRunningTask/platformfactory.cpp

```diff
@@ -0,0 +1,32 @@
+﻿#include "pch.h"
+
+#include <NotificationsLongRunningProcess_h.h>
+
+#include "platform.h"
+#include <wrl/implements.h>
+#include <wrl/module.h>
+#include "platformfactory.h"
+
+using namespace Microsoft::WRL;
+
+HRESULT NotificationsLongRunningProcessFactory::RuntimeClassInitialize()
+{
+    m_platform = Microsoft::WRL::Make<NotificationsLongRunningPlatformImpl>();
```

Comment:
You used WRL here because (IIRC) cppwinrt does not have complete support for outofproc COM servers?

Reply:
Yes